extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocation.java

```diff
@@ -623,58 +621,50 @@ private Pair<Object, Object> getMinMax(final Statistics<?> statistics) {
         return null;
     }
 
-    /**
-     * Apply the range filter to the row groups and return the result.
-     */
-    @NotNull
-    private PushdownResult pushdownRangeFilter(
+    private WritableRowSet removeExclusions(
```

Comment:
I'm not convinced there is a benefit to inverting the logic and returning unmatched rows. If you can point to a performance benefit, that's fine, but otherwise I prefer the creation and return of a PushdownResult.

Reply:
This is actually building the maybeMatch rows, not the exclusion. The reason why I've rmoved PushdownResult from these impl methods is b/c `PushdownResult#match` was _always_ empty, and I think it complicated the story of what was actually going on. The naming in some of these situations is weird; ie, instead of calling this `exclusions`, I'm calling this `removeExclusions`, but maybe that wasn't clear enough.